### PR TITLE
Features on screen saver control

### PR DESCRIPTION
**Pull request by sumibi-yakitori**
Since I'm currently using tetra to create tools rather than games, the
These features are necessary.

### CODE REVIEW

File: src/platform/window_sdl.rs [hunk 1]
```diff
@@ -355,6 +355,18 @@ impl Window {
             haptic.rumble_stop();
         }
     }
+
+    pub fn set_screen_saver_enable(&self, screen_saver_enable: bool) {
```
Comment (17cupsofcoffee) [inline src/platform/window_sdl.rs]:
```suggestion
    pub fn set_screen_saver_enabled(&self, screen_saver_enabled: bool) {
```

File: src/window.rs [hunk 2]
```diff
@@ -258,3 +258,13 @@ pub fn get_current_monitor_size(ctx: &Context) -> Result<(i32, i32)> {
     let monitor_index = ctx.window.get_current_monitor()?;
     ctx.window.get_monitor_size(monitor_index)
 }
+
+/// Set whether to enable the screen saver while the application is running.
+pub fn set_screen_saver_enable(ctx: &Context, screen_saver_enable: bool) {
```
Comment (17cupsofcoffee) [inline src/window.rs]:
```suggestion
pub fn set_screen_saver_enabled(ctx: &Context, screen_saver_enabled: bool) {
```

**Review by 17cupsofcoffee — APPROVED** [review]
Looks good, one suggestion with the function names but otherwise this seems good to go:

**Comment by sumibi-yakitori** [comment]
I am not a native speaker of English, so I appreciate this suggestion.

**Comment by 17cupsofcoffee** [comment]
Thank you for the PR 😄

**Pull request merged**